README.md

```diff
@@ -15,36 +24,54 @@ manipulate the data - even if the data is encrypted.
 
 To prevent these kinds of attacks the data must be encrypted in a tamper-resistant way.
 This means an attacker should not be able to:
- - Read the stored data - this is achieved by modern encryption algorithms.
+ - Distinguish the encrypted data from a random bit string - this is achieved by modern encryption algorithms.
  - Modify the data by changing parts of the encrypted data.
  - Rearrange or reorder parts of the encrypted data. 
 
-Authenticated encryption schemes (AE) - like AES-GCM or ChaCha20-Poly1305 - encrypt and
-authenticate data. Any modification to the encrypted data (ciphertext) is detected while
-decrypting the data. But even an AE scheme alone is not sufficiently enough to prevent all
-kinds of data manipulation.
+Authenticated encryption schemes (*AEAD*) - like AES-GCM or ChaCha20-Poly1305 - encrypt and
+authenticate data. Any modification to the encrypted data (*ciphertext*) is detected while
+decrypting the data. But even an *AEAD* scheme alone is not sufficiently enough to prevent all
```

Comment:
s/not sufficiently enough/insufficient

What does this statement mean "But even an *AEAD* scheme alone is not sufficiently enough to prevent all kinds of data manipulation in certain use cases"? If it is important to mention this, I think we should also specify details or at least point to a source of further information.